renovate.json

```diff
@@ -21,7 +21,8 @@
       "semanticCommitType": "fix",
       "group": true,
       "groupName": "required_provider",
-      "commitMessageExtra": "to latest for the landing zone DAs"
+      "commitMessageExtra": "to latest for the landing zone DAs",
+      "ignorePaths": ["patterns/vsi-extension/"]
     }
   ]
 }
```

Comment:
I just noticed that the `matchPaths` array on line 7 is missing `"patterns/vsi-extension"` - can you add it please?